Products/Zuul/infos/devicemanagement.py

```diff
@@ -39,7 +39,10 @@ def startTime(self):
 
     def updateWindow(self, p):
         self._object.manage_editMaintenanceWindow( 
-                                     startDateTime=p['startDateTime'],
+                                     startDateTime=p.get('startDateTime', None),
+                                     startDate=p.get('startDate', None),
+                                     startHours=p.get('startHours', None),
+                                     startMinutes=p.get('startMinutes', None),
```

Comment:
If startDateTime, startMinutes and startHours are not in the `p` map and get set to None, the manage_editMaintenanceWindow method will throw an exception.